src/main/java/io/reactiverse/pgclient/impl/RowImpl.java

```diff
@@ -45,6 +46,12 @@ public RowImpl(RowImpl row) {
     this.desc = row.desc;
   }
 
+  @Override
+  public String getColumnName(int pos) {
+    final ColumnDesc[] columnDescs = desc.columns();
+    return pos == -1 || columnDescs.length < pos ? null : columnDescs[pos].getName();
```

Comment:
I think if pos < 0 would be better to make sure of all negative numbers

Reply:
i am adding a negative test as well.